model-deployment/containers/nim/README-MODEL-CATALOG.md

```diff
@@ -0,0 +1,75 @@
+# Overview
+
+Utilising Model Catalog to store Models in OCI. We describe two ways to achieve this: 
+
+* Storing zipped model file in Model Catalog
+* Utilising Model-by-reference Feature to download model
+
+# Pre-requisite
+
+The following are the pre-requisite:
+* Notebook session with internet access (Recommended)
+* Download the Llama 3 8B Instruct Model from [HuggingFace](https://huggingface.co/meta-llama/Meta-Llama-3-8B-Instruct) or NGC repository.
+
+## Download NIM Container image and upload to OCIR
+* Pull the latest NIM Image to local machine or through NB session. Tag it with desired name.
+    ```bash
+    docker pull nvcr.io/nim/meta/llama3-8b-instruct:latest
+    docker tag nvcr.io/nim/meta/llama3-8b-instruct:latest odsc-nim-llama3:latest 
+    ```
+## OCI Container Registry
+
+Once NIM container is pushed, you can now use the `Bring Your Own Container` Deployment in OCI Data Science to deploy the Llama3 model.
+
+# Method 1: Export Model to Model Catalog
+
+Follow the steps mentioned [here](https://github.com/oracle-samples/oci-data-science-ai-samples/blob/main/model-deployment/containers/llama2/README.md), refer the section One time download to OCI Model Catalog. 
```

Comment:
We discussed adding specific sections Gaurav
You need to add hyperlink to the section, not the whole readme page